docs/blueprint-validation.md

```diff
@@ -12,10 +12,10 @@ project(s):
 One can [explicitly define validators](#explicit-validators), however, the
 expectation is that the implicit behavior will be useful for most users. When
 implicit, a validator is added if all deployment variables matching its inputs
-is defined. The `test_apis_enabled` validator is always enabled because it reads
-the entire blueprint and does not require any specific deployment variable. If
-`project_id`, `region`, and `zone` are defined as deployment variables, then the
-following validators are enabled:
+is defined. Validators listed below that have no inputs are always enabled
+because they read the entire blueprint and do not require any specific
+deployment variable. If `project_id`, `region`, and `zone` are defined as
+deployment variables, then the following validators are enabled:
 
```

Comment:
Addressed, although I added the list of validators (updated) to the "explicit validators" section because I believe it is useful to have a starting point from which to work.